go-controller/pkg/util/net.go

```diff
@@ -139,6 +139,23 @@ func JoinHostPortInt32(host string, port int32) string {
 	return net.JoinHostPort(host, strconv.Itoa(int(port)))
 }
 
+// SplitHostPortInt32 splits a vip into its host and port counterparts
```

Comment:
You could use: https://golang.org/pkg/net/#SplitHostPort ?

Reply:
done